tests/quantization/test_algo_quantization.py

```diff
@@ -194,22 +197,6 @@ def forward(self, x):
 #         \            \
 #         sigmoid - conv2d
 #
-def test_get_weight_activation_pairs__with_extra_module():
-    model_cls = DoubleWeightsPerActivationWithExtraModule
-    model_name = model_cls.__name__
-    config = get_quantization_config_without_range_init()
-    config["compression"].update({
-        "quantizable_subgraph_patterns": [["sigmoid", "conv2d"]],
-        "quantize_inputs": False})
-
-    _, algo = create_compressed_model_and_algo_for_test(model_cls(), config)
-
-    actual_pairs = algo.get_weights_activation_quantizers_pairs()
-    ref_pair_names = [(['NNCFConv2d[conv1]module_weight', 'NNCFConv2d[conv2]module_weight'],
-                       'ReLU[relu]/RELU_0')]
-
-    compare_weights_activation_quantizers_pairs(actual_pairs, algo, ref_pair_names, model_name)
-
```

Comment:
Why was this test removed?

Reply:
no need to remove it, until pattern-based mode removal

Reply:
Done.